test/system/executable/CMakeLists.txt

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########################################################################
+# Create tests                                                            #
+###########################################################################
+
+find_package(PythonInterp 3 REQUIRED)
+
+if(PYTHONINTERP_FOUND)
+
+    # Name of files to test
+    set(TEST_CONFIGURATIONS
+        void_configuration
+        simple_configuration
+    )
+
+    # Copy configuration files
+    # set(CONFIGURATION_FILES
+    #     "configurations/void_configuration.yaml"
+    #     "configurations/simple_configuration.yaml"
+    # )
+    foreach(TEST_CONFIGURATION ${TEST_CONFIGURATIONS})
+        configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configurations/${TEST_CONFIGURATION}.yaml
+            ${CMAKE_CURRENT_BINARY_DIR}/configurations/${TEST_CONFIGURATION}.yaml
+            COPYONLY)
+    endforeach()
+
+    # windows auxiliary script to fork test execution
+    # set(PWS_LAUNCHER
+    #     ${CMAKE_CURRENT_SOURCE_DIR}/launcher.ps1
+    # )
+
+    # Calculate environment
+    set(TEST_ENVIRONMENT
+        "PATH=$ENV{PATH};$<TARGET_FILE_DIR:fastcdr>;$<TARGET_FILE_DIR:fastrtps>;$<TARGET_FILE_DIR:ddsrouter>")
+
+    # if(WIN32)
+
+    #     if(TARGET tinyxml2 OR TARGET tinyxml2::tinyxml2)
+    #         set(TEST_ENVIRONMENT
+    #             "${TEST_ENVIRONMENT};$<TARGET_FILE_DIR:${TINYXML2_LIBRARY}>")
+    #     elseif(EXISTS TINYXML2_LIBRARY)
+    #         get_filename_component(
+    #             TINYXML2_LIBRARY_DIR ${TINYXML2_LIBRARY} DIRECTORY)
+    #         set(TEST_ENVIRONMENT
+    #             "${TEST_ENVIRONMENT};${TINYXML2_LIBRARY_DIR}")
+    #         unset(TINYXML2_LIBRARY_DIR)
+    #     endif()
+
+    #     string(REPLACE ";" "\\;" TEST_ENVIRONMENT "${TEST_ENVIRONMENT}")
+
+    # endif(WIN32)
+
+    # populate the tests
+    foreach(TEST IN LISTS TEST_CONFIGURATIONS)
+
+        set(TEST_NAME "system.ddsrouter.${TEST}")
+
+        # CTest has issues with signals on windows. We forked the test
+        # execution using an auxiliary powershell script
+        if(WIN32)
+        #    add_test(
```

Comment:
These tests are not running on windows

Reply:
done in commit https://github.com/eProsima/DDS-Router/pull/95/commits/d0f2c6aba020e393210a3512b03a865011891cc6